controllers/awsmachine_controller.go

```diff
@@ -822,12 +822,12 @@ func (r *AWSMachineReconciler) registerInstanceToClassicLB(machineScope *scope.M
 
 	if err := elbsvc.RegisterInstanceWithAPIServerELB(i); err != nil {
 		r.Recorder.Eventf(machineScope.AWSMachine, corev1.EventTypeWarning, "FailedAttachControlPlaneELB",
-			"Failed to register control plane instance %q with load balancer: %v", i.ID, err)
+			"Failed to register control plane instance %q with classic load balancer: %v", i.ID, err)
```

Comment:
This is delibarately not mentioning `classic` because once the "classic" load balancer expires, we don't have to go around and change all the log messages. :) Besides, from the user's perspective, it shouldn't matter. It's just a load balancer.